### PR TITLE
Updated return value of get_loss functions for correctness

### PR DESCRIPTION
**Pull request by fhieber**
Previously CrossEntropy and SmoothedCrossEntropy had different return types.
Their return values are grouped in training.py::121, which expects a list as input.

### CODE REVIEW

File: sockeye/loss.py [hunk 1]
```diff
@@ -68,24 +68,24 @@ class CrossEntropyLoss(Loss):
...
         :param labels: Shape: (batch_size * target_seq_len,).
-        :return: Loss and softmax output symbols.
+        :return: List of loss and softmax output symbols.
         """
         if self._normalize:
             normalization = "valid"
         else:
             normalization = "null"
-        return mx.sym.SoftmaxOutput(data=logits,
+        return [mx.sym.SoftmaxOutput(data=logits,
```
Comment (davvil) [inline sockeye/loss.py]: I cannot find any clear documentation on this. What does SoftmaxOutput return exactly? If it's a tuple, then here we are creating a list with a single (tuple) element.

Comment (fhieber) [inline sockeye/loss.py]: Its returning a single symbol

Comment (davvil) [inline sockeye/loss.py]: I thought so. Then the :return: specification is not correct, is it?

Comment (fhieber) [inline sockeye/loss.py]: updated
